Disable numeric_limits for floating point types when included with -DKERNEL
--- gcc-4.9.1.orig/libstdc++-v3/include/std/limits	2014-01-02 23:30:10.000000000 +0100
+++ gcc-4.9.1/libstdc++-v3/include/std/limits	2014-10-22 16:35:21.415179603 +0200
@@ -1548,6 +1548,7 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
     };
 #endif
 
+#ifndef KERNEL
   /// numeric_limits<float> specialization.
   template<>
     struct numeric_limits<float>
@@ -1772,6 +1773,7 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
 #undef __glibcxx_long_double_has_denorm_loss
 #undef __glibcxx_long_double_traps
 #undef __glibcxx_long_double_tinyness_before
+#endif // KERNEL
 
 _GLIBCXX_END_NAMESPACE_VERSION
 } // namespace
